ansible/group_vars/all

```diff
@@ -387,3 +387,26 @@ metrics:
     port: "{{ metrics_kamon_statsd_port | default('8125') }}"
 
 user_events: "{{ user_events_enabled | default(false) }}"
+
+etcd:
+  version: "{{ etcd_version | default('v3.3.12') }}"
+  client:
+    port: 2379
+  server:
+    port: 2480
+  cluster:
+    token: "{{ etcd_cluster_token | default('openwhisk-etcd-token') }}"
+  dir:
+    data: "{{ config_root_dir }}/etcd"
+  lease:
+    timeout: "{{ etcd_lease_timeout | default(1) }}"
+  quota_backend_bytes: "{{ etcd_quota_backend_bytes | default(0) }}"
+  snapshot_count: "{{ etcd_snapshot_count | default(100000) }}"
+  auto_compaction_retention: "{{ etcd_auto_compaction_retention | default(0) }}"
```

Comment:
`retetion` should not be 0?

Reply:
`retention == 1` is more better. I will update it :)